bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+        except Exception as AuthError:
```

Comment:
@cverna , nope its not working for the overrides, comments and request. I found the solution for the request but stuck in case of comments and overrides.